ScoutSuite/frontend/src/components/Partial/PartialValue/index.js

```diff
@@ -54,13 +55,14 @@ const PartialValue = props => {
     tooltip,
     tooltipProps,
     renderValue,
+    baseErrorPath
   } = props;
 
   const ctx = useContext(PartialContext);
   const basePath = useContext(PartialPathContext);
   const setIssueLevel = useContext(PartialTabContext);
 
-  const fullValuePath = concatPaths(basePath, valuePath);
+  const fullValuePath = concatPaths(typeof baseErrorPath !== 'undefined' ? baseErrorPath  : basePath, valuePath);
```

Comment:
is it intended to mix `baseErrorPath` with `fullValuePath`, I would think the prop would be used for when the path prefix is different for the value and for the error, so you'd want a way to specify the base path for each of them.